packages/boot/test/bootstrapTests/supports.js

```diff
@@ -197,7 +203,7 @@ export const getNodeTestVaultsConfig = async (
   assert(config);
 
   // speed up (e.g. 80s vs 133s with xs-worker in production config)
-  config.defaultManagerType = 'local';
+  config.defaultManagerType = defaultManagerType;
```

Comment:
this comment only applies to `local`

Reply:
Good catch.

(Comments drifting out of consistency with the things they are commenting on is one of those persistent code problems that compilers just can't notice.  Maybe AI tools in the future will be able to.)